Update VSCode Wizard documentation (#1889)
telerik/blazor-docs: getting-started/vs-code-integration/new-project-wizard.md

1. Type/Select `Telerik UI for Blazor Template Wizard: Launch` and press `Enter` to launch the extension.

## Creating the Project

The **Create New Project** wizard provides two options to start you project:

> * **Start from Blank Project** - allows you to create a blank project that is pre-configured for the Progress® Telerik® UI for Blazor components. You can select and add sample pages to get started (a Grid with CRUD operations form a service, a Form, Charts).

Reviewer: In the screenshot, it's `Start from Blank App`.

> * **Start from a Template** - the wizard provides a set of predefined project templates:

Reviewer: Double-check the screenshot, it says "Start from Template"

```suggestion
* **Start from Template** - the wizard provides a set of predefined project templates:
```

>     * **CRUD, Form, Chart**&mdash;A small app that showcases the Telerik UI Data Grid, Chart, and forms validation. The Grid also uses a basic CRUD service which is a common pattern for data updates in real applications.

Reviewer: * Should it be `Form validation` instead?
* For the second sentence, consider going with `The Data Grid also uses a CRUD service (a common pattern for data updates in production-ready applications).`

>     * **Dashboard**&mdash;A basic dashboard layout that uses the TileLayout component for customizable dashboards, and showcases several ways individual blocks (tiles) can fetch data.

Reviewer: ```suggestion
    * **Dashboard**&mdash;A basic dashboard that showcases the Telerik UI TileLayout component and how individual blocks (tiles) can fetch data.
```

>     * **Admin**&mdash;A small app that shows a dashboard for admin. The app uses some of the main Telerik UI components like the [Data Grid]({%slug grid-overview%}), [Chart]({%slug components/chart/overview%}), [TileLayout]({%slug tilelayout-overview%}), [Form]({%slug form-overview%}), and more.

Reviewer: ```suggestion
    * **Admin**&mdash;A small app that showcases a dashboard for administrators. The app uses some of the main Telerik UI components like the [Data Grid]({%slug grid-overview%}), [Chart]({%slug components/chart/overview%}), [TileLayout]({%slug tilelayout-overview%}), [Form]({%slug form-overview%}), and more.
```

## Configuring the Project

> The wizard allows you to configure the following options:

Reviewer: Is this for both options (blank app and template), including all template sub-options? If this is the case, consider going with `In the **Telerik UI for Blazor Template Wizard**, you can configure the following options for your new Blazor app:**

> * Location and name for your app.

Reviewer: Go with what you're seeing in the screenshot. In this case, it seems to be `Project name and output path.` If these are not native options to other VSCode wizards, consider providing more detail.

> * Hosting model type - [WebApp]({%slug getting-started/web-app%}), [WebAssembly]({%slug getting-started/client-side%}), [Server-Side]({%slug getting-started/server-side%}) or [Hybrid Blazor MAUI]({%slug getting-started/hybrid-blazor%}) app.

Reviewer: Go with what people will see in the screenshot. The current screenshot says WebApp, Client, Server, and Hybrid. Provide additional details in parenthesis. Also, the names of UI options, menus, etc. are usually formatted with bold. For example:
```suggestion
* Hosting model type&mdash;[**WebApp**]({%slug getting-started/web-app%}), [**Client** (also: WebAssembly)]({%slug getting-started/client-side%}), [**Server**]({%slug getting-started/server-side%}), or [**Hybrid** (a hybrid Blazor MAUI)]({%slug getting-started/hybrid-blazor%}) app.
```

> * Your Telerik license (trial or commercial, depending on what you have). 

Reviewer: I would use the opportunity to briefly address paid vs trial. Do they need to set up their license somehow, do they need to do anything special if they choose paid, etc. For example: 
```suggestion
* Your Telerik license. If set to **Trial**, you will see a trial watermark and banner over your application. If set to **Paid**, you need to ... 
```

* Whether to use localization in the project or not.

> * Whether to use font icons in the project or not.

Reviewer: In a future version, consider reworking the UI for this. The Icon Settings label for the button is not intuitive or user-friendly. I would probably switch the labels: `Configure icons` with the button label being `Use font icons`.

Author: This depends on the team that manages the VSCode extension. Maybe you can open an issue with the suggestion or discuss it with a member of the team.

> * The .NET version you want to use.

Reviewer: Can people choose a different .NET version? Does it depend on what they have installed on their system? If the latter, I would add a line saying `The wizard shows only the .NET versions available on your system.`

Author: The wizard provides a general .NET version configuration option regardless of what is installed on one's machine.

Reviewer: If that's the case, I'd argue about the merit of a screenshot showing how people can choose from one possible option.

> * Sample pages that you can add to get started (available for `Start from Blank Project` option).

Reviewer: ```suggestion
* Sample pages that you can add to get started (available when `Start from Blank App` is selected).
```
